packages/app/src/app/pages/Dashboard/Content/routes/Recent/RecentHeader.tsx

```diff
@@ -101,16 +97,11 @@ export const RecentHeader: React.FC<{ title: string }> = ({ title }) => {
 };
 
 const TopBanner = () => {
-  const { ubbBeta } = useWorkspaceFeatureFlags();
   const [welcomeBannerDismissed] = useDismissible('DASHBOARD_UBB_BETA_WELCOME');
 
-  if (ubbBeta) {
-    if (welcomeBannerDismissed) {
-      return <UbbUpgradeBanner />;
-    }
-
-    return <UBBBetaWelcomeBanner />;
+  if (welcomeBannerDismissed) {
+    return <UbbUpgradeBanner />;
   }
 
-  return <UpgradeBanner />;
+  return <UBBBetaWelcomeBanner />;
```

Comment:
Drop the Beta name? 😄

Reply:
@danilowoz is working on this in a separate branch